api/v1/configurationpolicy_types.go

```diff
@@ -27,6 +27,10 @@ type RemediationAction string
 // +kubebuilder:validation:Enum=low;Low;medium;Medium;high;High;critical;Critical
 type Severity string
 
+// PruneObjectBehavior : DeleteAll, DeleteIfCreated, or None
+// +kubebuilder:validation:Enum=DeleteAll;DeleteIfCreated;None;
+type PruneObjectBehavior string
```

Comment:
Optional: What do you think about adding a default value of `None` explicitly with a kubebuilder marker?
https://book.kubebuilder.io/reference/markers/crd-validation.html#crd-validation

I think this does two things:
* It makes it clear what the pruneObjectBehavior is regardless of if it was explicitly set.
* It makes the new field discoverable when existing policies are queried from the API.

Reply:
sure, I can add this too